apps/storybook-react/.storybook/main.js

```diff
@@ -19,4 +25,34 @@ module.exports = {
       },
     },
   ],
+  webpackFinal: async (config, { configType }) => {
```

Comment:
Where is the value for `configType` set/decided?

Reply:
Read the docs